project/perform-release.sh

```diff
@@ -9,10 +9,15 @@ if [[ $1 != "patch" && $1 != "minor" && $1 != "major" ]] ; then
   exit 1;
 fi
 
-echo ""
-echo "Sign into npm"
-echo ""
-npm whoami &>/dev/null || npm login
+currentBranch="$(git symbolic-ref HEAD 2>/dev/null)" ||
```

Comment:
Looks weird over two lines. Could you fit it on one line with a shorter variable name?

Actually, since you bail if the branch isn't "master", the second part could just be `currentBranch=""` and have the same effect. What does the first part set the variable to if you are in a detached head state, anyway?